src/CosmosDB/CosmosDB/CosmosDBAccount/NewAzCosmosDBAccount.cs

```diff
@@ -85,6 +85,10 @@ public class NewAzCosmosDBAccount : AzureCosmosDBCmdletBase
         [PSArgumentCompleter("Sql", "MongoDB", "Gremlin", "Cassandra", "Table")]
         public string ApiKind { get; set; }
 
+        [Parameter(Mandatory = false, HelpMessage = Constants.PublicNetworkAccessHelpMessage)]
+        [ValidateNotNullOrEmpty]
+        public string PublicNetworkAccess { get; set; }
```

Comment:
Are the values of this string fixed ?? If yes, then use PSArgumentCompleter, (see ApiKind for reference)

Reply:
added PSArgumentCompleter. Thanks